masterlist.yaml

```
@@ -9547,6 +9547,95 @@ plugins:
+      - name: 'Kapotun ARMA Moonpath.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA NFF.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA PracticalPirate.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TAWOBA.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TBD Bikini.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TBDUnique Treasures.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Unique Uniques.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA USMP.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA USSEP.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA WetAndCold.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Winterhold Restored.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Wyrmstooth.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun SOS.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun Vanilla Armor Patch.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
```

The first version check is sufficient. The original version doesn't have any version metadata, so comparing it to a given version with `==`, `>`, or `>=` will always return false, & the incompatibility won't be displayed. The rest of the condition is unnecessary & will actually always evaluate to false as that specific checksum of the plugin has no version metadata, so it is less than 2.0.0:
```suggestion
      - name: 'Kapotun SOS.esp'
        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=)'
      - name: 'Kapotun Vanilla Armor Patch.esp'
        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=)'
```